mpl_gui/__init__.py

```diff
@@ -219,20 +253,27 @@ def close_all(self):
         passing it to `show`.
 
         """
-        for fig in self.figures:
-            if fig.canvas.manager is not None:
-                fig.canvas.manager.destroy()
+        for fig in list(self.figures):
+            self.close(fig)
+
+    def close(self, val):
```

Comment:
Note this doesn't allow mgr.close() to close the current figure.  If we are going to force the user to keep a pointer to `fig` around in order to close figures, then it seems `fig.close()` is a more natural home for this functionality?

Reply:
We do not yet have a notion of "current figure", but using a pattern like we do with `Artist.remove` is probably the right thing to do for `Figure.close`

Reply:
I created #8 to track this.